apps/community/backend/cmd/api/main.go

```diff
@@ -79,50 +57,68 @@ func KeyGen() string {
  TODO: 	create CommunityConfig struct which contains globals for the community like
 		swarm key and name of it and peer ids in it
 */
-func CreateCommunity(name string, path string) (error, string, string) {
+func CreateCommunity(name string, path string) (error, string, string, []string) {
 
+	root := os.Getenv("ROOT")
```

Comment:
Stuff like this can use the compass package in my current PR.

Reply:
I'll wait til you merge it then rebase?